packages/graphql/src/schema/generation/where-input.ts

```diff
@@ -37,6 +37,8 @@ import { withAggregateInputType } from "./aggregate-types";
 import { augmentWhereInputTypeWithRelationshipFields } from "./augment-where-input";
 import { makeImplementationsWhereInput } from "./implementation-inputs";
 
+const isEmptyObject = (obj: Record<string, unknown>) => !Object.keys(obj).length;
```

Comment:
purely for consistency, please change this to a `function` syntax

```suggestion
function isEmptyObject(obj: Record<string, unknown>): boolean {
    return !Object.keys(obj).length;
}
```